my-app/src/app/core/services/auth.service.mock.ts

```diff
@@ -0,0 +1,23 @@
+import {ILocalStorageUserInfo} from "@shared/";
+
+interface IAuthService {
+  login(): void,
+
+  logout(): void,
+
+  getUserInfo(): ILocalStorageUserInfo;
+}
+
+export class AuthServiceMock implements IAuthService {
+  private tokenString: string = 'sssfjn28295wsfssfnfsf092425f';
```

Comment:
I don't see any reason to have private members in mocks

Reply:
Fixed into pr https://github.com/Russzh/angular_course/pull/6